src/plugins/discover/public/application/components/doc_viewer/doc_viewer.scss

```diff
@@ -54,6 +53,35 @@
   align-items: flex-start;
 }
 
+.kbnDocViewer__table {
+  table {
+    table-layout: auto;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  .kbnDocViewer__table {
+    table {
+      table-layout: auto;
+    }
+  }
+
+  .kbnDocViewer__tableRow {
+    td:nth-of-type(1) {
+      max-width: 10%;
+    }
+
+    td:nth-of-type(2) {
+      max-width: 30%;
+      width: 30%;
+    }
+
+    td:nth-of-type(3) {
+      max-width: 60%;
+    }
```

Comment:
the problem with the CSS approach is, we are using the doc viewer in 2 ways, one with action column, and one without. So it would work incorrectly in the second case:
<img width="2282" alt="Bildschirmfoto 2021-11-02 um 11 54 33" src="https://user-images.githubusercontent.com/463851/139834224-e8b7aafc-403f-49be-bdfa-1346aed6fcef.png">
So I think the better approach would defining the width of the field and the value column in the EuiInMemoryTable column:


<img width="981" alt="Bildschirmfoto 2021-11-02 um 12 00 26" src="https://user-images.githubusercontent.com/463851/139834650-4c2f8a44-b5b9-4b9b-b47a-81cf0a5f62fc.png">

https://elastic.github.io/eui/#/tabular-content/in-memory-tables